### PR TITLE
Issue #636: Mention oc login at the end of oc-env

### PR DESCRIPTION
**Pull request by cfergeau**
This fixes https://github.com/code-ready/crc/issues/636

### CODE REVIEW

File: cmd/crc/cmd/oc_env.go [hunk 1]
```diff
@@ -23,16 +23,17 @@ var ocEnvCmd = &cobra.Command{
...
-		if proxyConfig.IsEnabled() {
-			output.Outln(shell.GetEnvString(userShell, "HTTP_PROXY", proxyConfig.HttpProxy))
-			output.Outln(shell.GetEnvString(userShell, "HTTPS_PROXY", proxyConfig.HttpsProxy))
-			output.Outln(shell.GetEnvString(userShell, "NO_PROXY", proxyConfig.GetNoProxyString()))
+		if clusterConfig.ProxyConfig.IsEnabled() {
+			output.Outln(shell.GetEnvString(userShell, "HTTP_PROXY", clusterConfig.ProxyConfig.HttpProxy))
+			output.Outln(shell.GetEnvString(userShell, "HTTPS_PROXY", clusterConfig.ProxyConfig.HttpsProxy))
+			output.Outln(shell.GetEnvString(userShell, "NO_PROXY", clusterConfig.ProxyConfig.GetNoProxyString()))
 		}
+		output.Outf("echo \"You can now login to the OpenShift cluster with 'oc login -p developer -u developer %s'\"\n", clusterConfig.ClusterAPI)
```
Comment (praveenkumar) [inline cmd/crc/cmd/oc_env.go]: @cfergeau only for developer user or also for admin?

Comment (cfergeau) [reply cmd/crc/cmd/oc_env.go]: Not sure we should advertise the admin user too strongly?

Comment (praveenkumar) [reply cmd/crc/cmd/oc_env.go]: @cfergeau most of the user who want to experiment with operators like making any config changes or installing a new one require the the admin even with operator hub so I think it should be good to have both user mentioned here like we do for `crc console credentials`.

Comment (cfergeau) [reply cmd/crc/cmd/oc_env.go]: But if we push too much the admin credentials, won't crc users always use that, and just ignore the `developer` user?

**Comment by cfergeau** [comment]
Ignore the commits I pushed, I did these by mistake ;)

File: cmd/crc/cmd/oc_env.go (hunk 1 above)
Comment (praveenkumar) [reply cmd/crc/cmd/oc_env.go]: @cfergeau it's just user choice we should provide them both the options like we are providing for other user facing commands.

**Pull request closed without merging**